src/core/engines/cross-margin/CrossMarginMath.sol

```diff
@@ -99,14 +108,15 @@ library CrossMarginMath {
 
     /**
      * @notice get minimum collateral
+     * @dev detail is composed of positions with the same underlying + strike + collateral + expiry
      * @param _detail margin details
-     * @return cashNeeded with {collateral asset's} decimals
+     * @return numeraireNeeded with {collateral asset's} decimals
      * @return underlyingNeeded with {underlying asset's} decimals
      */
```

Comment:
what is `numeraireNeeded` vs `underlyingNeeded`? 
It's unclear which asset they are denominated in.
If collateral is underlying, does it return the same value?

Reply:
Seems like `numeraireNeeded` should be denominated in USDC?

Reply:
Numeraire is in fact the better term here. 

Grappa is overloading the term `strike` to also describe the asset. 

In ETH / USDC, underlying is ETH and numeraire is USDC.

> If collateral is underlying, does it return the same value?
It certainly would not return the same value. Calls are collateralized in underlying, puts are collateralized in numeraire. In CrossMarginDetail we are collecting all the puts and calls that have the same underlying, numeraire (strike) asset, and expiry. Based on this combination we can cross margin the options.

If there are no put options, then the whole thing is assumed to be collateralized in underlying. If there is numeraire required to be posted (example call spread) then that amount is converted into equivalent underlying.

Reply:
I did, in the last commit, update the CrossMarginDetail struct to reflect what it actually should have been this whole time.